src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,77 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int CAPACITY = 10;
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```

It's initialized by 0 by default